pyaci/core.py

```diff
@@ -128,7 +128,9 @@ def _performRequest(self, method, format=None, needData=False, **kwargs):
 
         req = Request(method, url, data=data)
         prepped = rootApi._session.prepare_request(req)
-        self._x509Prep(rootApi, prepped, data)
+        # never use certificate for subscription requests
```

Comment:
You're right it might not be the safest assumption.  We know that we can't use signing when making the subscription GET request (403 error raised by the APIC).  Also, once the token refresh is added there will be scenarios where we have a valid token from requestAppToken API but need to use signing for GET requests to refresh the token.  

We could possibly add appcenter flag to the Node object and then within _x509Prep could check that flag in rootApi and if set only perform signing on aaaRequestToken APIs?

Reply:
Okay, let's leave this change like this for now. Let's comeback and refactor/reorganize later.